<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="b3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>90</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="b5">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>92</y>
     <width>90</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="b6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>92</y>
     <width>90</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="b7">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>164</y>
     <width>90</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="b9">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>164</y>
     <width>90</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="b4">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>92</y>
     <width>90</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="b8">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>164</y>
     <width>90</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="b2">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>20</y>
     <width>90</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Player 1 : X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Player 2 : O</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Play again </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Marker Felt</family>
     <pointsize>24</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Winner is </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Marker Felt</family>
     <pointsize>36</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="b1">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>20</y>
     <width>90</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
